### PR TITLE
Add CI script for documentation

### PR DESCRIPTION
**Pull request by DIOHz0r**
### Changes description

I've noticed that after generated RC2 the documentation for master wasn't updated because the script change was not on the release branch. This must fix the issue

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@DIOHz0r |2|

### References

Closes #N/A
Related #N/A
Depends on #N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,6 +57,7 @@ script:
 after_success:
   # let's update the documentation and locales
   - if [ ${TRAVIS_PHP_VERSION:0:3} == "$CS" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then tests/after_success.sh; fi
+  - if [ ${TRAVIS_PHP_VERSION:0:3} == "7.1" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then tests/apidocs.sh; fi
```
Comment (btry) [inline .travis.yml]: prefer a variable like we do with $CS, this is more maintainable

Comment (DIOHz0r) [reply .travis.yml]: This is the only place used for check the PHP 7.1 version, is really necessary?

Comment (btry) [reply .travis.yml]: this is for convenience, when php will raise its version. This also gives the opportunity to comment the variable and tell it must be different from $CS to ensure apigen does not conflicts with other dependencies in composer.json

**Comment by btry** [comment]
@DIOHz0r 

I think we can / should cherry-pick this pr into develop. Waiting for your approval.

**Pull request merged**